docs/cadence/language/values-and-types.mdx

```diff
@@ -1116,7 +1116,7 @@ are available for both variable-sized and fixed-sized or variable-sized arrays.
 
 -
     ```cadence
-    fun concat(_ array: T): T
+    access(all) fun concat(_ array: T): T
```

Comment:
Maybe consider adding a line break after the access modifier here and for the other functions below

Reply:
updated in 8d8fd5c116e1d50f68e45f4a5b9c03543103c46e